src/metabase/models/revision/diff.clj

```diff
@@ -76,11 +76,9 @@
     [:dataset_query _ _]
     (deferred-tru "modified the query")
 
-    [:dataset false true]
-    (deferred-tru "turned this into a model")
-
-    [:dataset true false]
-    (deferred-tru "changed this from a model to a saved question")
+    ;; report_card.type
+    [:type old new]
+    (deferred-tru "type changed from {0} to {1}" old new)
```

Comment:
You can keep the old behaviour here by matching on the values e.g. `[:type "model" "question"]`

Reply:
good call https://github.com/metabase/metabase/pull/38074/commits/9c2d57c5947e77e1494fe654c95c0448a898f4ee